<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>healthassessment</class>
 <widget class="QWidget" name="healthassessment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#healthassessment{border-image: url(:/back/蓝色渐变.jpg);}
</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>167</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit{border-radius:5px;}
#lineEdit{background-color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="placeholderText">
    <string>请输入你的身高/m</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>167</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_2{border-radius:5px;}
#lineEdit_2{background-color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="placeholderText">
    <string>请输入你的体重/kg</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>231</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_3{border-radius:5px;}
#lineEdit_3{background-color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="placeholderText">
    <string>请输入你的收缩压/mmHg</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>298</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_4{border-radius:5px;}
#lineEdit_4{background-color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="placeholderText">
    <string>请输入你的TC值(mmol/L)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>298</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_5{border-radius:5px;}
#lineEdit_5{background-color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="placeholderText">
    <string>请输入你的空腹血糖(mg/dl)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>230</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_6{border-radius:5px;}
#lineEdit_6{background-color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="placeholderText">
    <string>请输入你的舒张压/mmHg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton{border-radius:10px;}
#pushButton{background-color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>447</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2{border-radius:10px;}
#pushButton_2{background-color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>374</y>
     <width>431</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tableWidget{border-radius:5px;}
#tableWidget{background-color: rgb(255, 255, 255);}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>健康评估</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>healthassessment</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
